conda_smithy/configure_feedstock.py

```diff
@@ -1731,6 +1731,7 @@ def _load_forge_config(forge_dir, exclusive_config_file, forge_yml=None):
             "win_64": "win_64",
             "osx_64": "osx_64",
         },
+        "noarch_platform": "linux_64",
```

Comment:
Should this be `noarch_platforms` with an `s`?

Reply:
was emulating `build_platform`, etc. i think multiple will actually be rare...

Reply:
bc22a0036

Reply:
Yeah, `noarch_platform` might be better in that case.